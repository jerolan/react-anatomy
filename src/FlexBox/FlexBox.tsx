import styled from 'styled-components';

const FlexBox = styled.div`
  display: flex;
`;

export default FlexBox;
