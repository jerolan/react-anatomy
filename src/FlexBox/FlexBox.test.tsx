import React from 'react';
import renderer from 'react-test-renderer';

import FlexBox from './FlexBox';

describe('<FlexBox />', () => {
  it('renderenders without crashing', () => {
    const component = renderer.create(<FlexBox />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
