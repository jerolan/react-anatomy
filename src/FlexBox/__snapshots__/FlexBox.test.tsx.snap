// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlexBox /> renderenders without crashing 1`] = `
<div
  className="sc-bdVaJa kZAgjJ"
/>
`;
